blocks/form/form.css

```diff
@@ -159,7 +159,9 @@ main .form-submit-wrapper .button {
   color: black;
 }
 
-main .form input[type="text-field"] {
+main .form input[type="text-field"],
```

Comment:
Sorry, I am new to the project but there is no type `text-field`. I think you mean `text`

[1] https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#input_types

Reply:
Yes

Reply:
text-field not issue its means text please review